docs/how-to/remote-debugging-dotnet.md

```diff
@@ -0,0 +1,5 @@
+## How to setup remote debugging of dotnet Azure Functions on new deployments
+
+1) when running `deploy.py` use `--host_dotnet_on_windows` flag as part of the command line. This will deploy `dotnet` Azure Function on Windows Server Farm, which supports functinoality for remote debugging of `dotnet` code.
```

Comment:
```suggestion
1) When running `deploy.py` use `--host_dotnet_on_windows` flag as part of the command line. This will deploy `dotnet` Azure Function on Windows Server Farm, which supports functionality for remote debugging of `dotnet` code.
```